Spring 2026 roadmap update blog post (#479)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: blog/2026-04-27.md

---
title: 'Ed-Fi Release Roundup: Spring and Summer 2026'
authors: [fuqua]
tags: [releases, data-standard, ods-api, oneroster, dms, admin-app]
date: 2026-04-27
---

A lot is shipping over the next few months. If you're planning a deployment cycle or tracking what's coming in Ed-Fi infrastructure, here's a practical overview of what to expect and why it matters.

## May Releases: Four at Once

> Four significant releases are targeting May: **Data Standard 6.1**, **ODS/API 7.3.2**, **MetaEd**, and the **Ed-Fi OneRoster API**. Each one stands on its own, but they're designed to work together.

Reviewer: ```suggestion
Four significant releases are targeting May: **Data Standard 6.1**, **ODS/API 7.3.2**, **MetaEd 4.7**, and the **Ed-Fi OneRoster API**. Each one stands on its own, but they're designed to work together.
```

### MetaEd v4.7

**MetaEd** v4.7 adds support for Data Standard v6.1; since MetaEd is the tool that implementers use to explore and understand the Data Standard, it needs to reflect these changes before they can be widely adopted. Additionally, it contains bug fixes and a new API Catalog spreadsheet artifact that provides a comprehensive reference for all API endpoints.

### Ed-Fi OneRoster API

The [**Ed-Fi OneRoster API**](./2026-01-30.md) provides a standards-based roster data interface — compatible with both Data Standard v4 and v5 — built on the same authorization model as the ODS/API. It supports both SQL Server and PostgreSQL, which removes a constraint that would have blocked some shops from adopting it.

This gives Ed-Fi adopters a path to satisfy OneRoster API 1.2 integration requirements without standing up a separate roster service that's disconnected from the rest of the Ed-Fi stack.

## Later This Year

### Ed-Fi API v8 — July

> The **Ed-Fi API v8** (formerly "Data Management Service 1.0") release is the milestone that the project has been building toward. we will have more to say once we are closer to the release date, but this is the production-ready version of the next-generation Ed-Fi API platform.

Reviewer: ```suggestion
The **Ed-Fi API v8** (formerly "Data Management Service 1.0") release is the milestone that the project has been building toward. We will have more to say once we are closer to the release date, but this is the production-ready version of the next-generation Ed-Fi API platform.
```

## Data Import: Final Release

The final release of **Data Import** ([v2.4](/reference/data-import)) shipped recently, with .NET 10 support added. No new features are planned — this closes out the product. If you're running Data Import today, this release keeps you on a supported runtime. If you're evaluating ETL tooling for new implementations, plan around alternatives; Data Import is not the path forward.

---

The pace of releases this spring and summer reflects real momentum — particularly around the Ed-Fi API v8 and the operational tooling improvements in Admin App v4.1 and ODS Admin API v2.4. If your team is planning upgrades or new deployments, now is a good time to align your timelines with these release windows. I'd especially watch the Ed-Fi API v8 release in July; that's the one with the longest-term implications for how Ed-Fi infrastructure is built and operated.

:::tip

> For more detail on upcoming releases, checkout the [Product Roadmap](https://github.com/orgs/Ed-Fi-Alliance-OSS/projects/2) in GitHub. It's a living document that tracks planned features, release timelines, and the status of ongoing work across the Ed-Fi ecosystem.

Reviewer: ```suggestion
For more detail on upcoming releases, check out the [Product Roadmap](https://github.com/orgs/Ed-Fi-Alliance-OSS/projects/2) in GitHub. It's a living document that tracks planned features, release timelines, and the status of ongoing work across the Ed-Fi ecosystem.
```

Author: My spelling capabilities have been corrupted by `git` 😆
